pkg/util/labels_test.go

```diff
@@ -0,0 +1,35 @@
+package util
+
+import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"testing"
+)
+
+func TestImageMetadataLabels(t *testing.T) {
```

Comment:
@gabemontero can we get an e2e in our integration tests - ideally for both the `docker` and `dockerfile` flows.

Reply:
ah ... those are new since I last traversed these waters

sure, I'll add an `image_metadata.json` file in the `.s2i` folder and add the labels there to the expected contents

Reply:
quick update @adambkaplan - there appears to be a image_metadata.json test via https://github.com/openshift/source-to-image/blob/master/test/integration/scripts/.s2i/bin/assemble#L19-L24 but based on my examinations that only applies to the `docker` integration tests, and not the `dockerfile` ones (which I think explains the gap here).

Reply:
@adambkaplan and I talked and since even with dockerfile, we have to git clone a repo with a image metadata file and there is no injection path for it like the scripts, I'm just going to broaden the new unit test a bit.